packages/legend-extension-dsl-diagram/src/DiagramRenderer.ts

```diff
@@ -1630,7 +1660,7 @@ export class DiagramRenderer {
     // NOTE: force hash reload when we redraw class view; this would help with cases where
     // we auto add new properties to the class view, causing the box to expand, hence we need
     // to recompute hash
-    classView.forceRefreshHash();
+    positionedRectangle_forceRefreshHash(classView);
```

Comment:
We should have a flag to disable/enable change detection in diagram renderer to control this behavior and `action` modifier methods